.circleci/config.yml

```diff
@@ -4,21 +4,12 @@ jobs:
     working_directory: ~/circle
     docker:
       - image: circleci/ruby:2.6.4
-        environment:
-          RAILS_ENV: test
-          PGHOST: 127.0.0.1
-          PGUSER: root
-          DATABASE_URL: postgresql://root@localhost/fb-submitter_test
-      - image: circleci/postgres:10.6-alpine-ram
-        environment:
-          POSTGRES_USER: root
-          POSTGRES_DB: circle-test_test
     steps:
       - checkout
       - setup_remote_docker
       - run:
           name: test
-          command: 'make spec'
+          command: docker-compose run --rm app bundle exec rspec
```

Comment:
Why not `make spec`? Seems better to always use just one command everywhere

Reply:
1. `make spec` mounts the filesystem now (not supported by circle)

2. Decouples our deploys from a makefile. Devs are now free to do what they see fit in the makefile without the side effect of braking circle ci. IMO makefiles are a convenience for developers and only alias calls. having circle call them seems like a reverse dependency of this

Reply:
👍